src/systems/filecoin_markets/deal/_index.md

```diff
@@ -65,3 +65,34 @@ On every PoSt Submission, the following steps happen.
 # Deal Flow
 
 {{< diagram src="diagrams/deal-flow.mmd.svg" title="Deal Flow Sequence Diagram" >}}
+
+# Deal States
+
+All on-chain economic activities in Filecoin start with the deal. This section aims to explain different states of a deal and their relationship with other concepts in the protocol such as Power, Payment, and Collaterals.
+
+A deal has the following states:
+
+- `Unpublished`: the deal has yet to be posted on chain.
+- `Published`: the deal has been published and accepted by the chain but is not yet active as the sector containing the deal has not been proven.
+- `Active`: the deal has been proven and not yet expired.
+- `Deleted`: the deal or the sector containing the deal has expired or the sector containing the deal has been terminated because of faults.
+
+Note that `Unpublished` and `Deleted` states not tracked on chain. To reduce on-chain footprint, an `OnchainDeal` struct is created when a deal is published and it keeps track of a `LastPaymentEpoch` which defaults to zero when a deal is in the `Published` state. A deal transitions into the `Active` state when `LastPaymentEpoch` is positive.
```

Comment:
when `LastPaymentEpoch` is positive and `ExpirationDate` is in the future?

Reply:
We are using whether `LastPaymentEpoch` is positive to determine if a deal is Active but If `EndEpoch` (ExpirationDate) is not in the future, it will fail `ActivateDeal` and result in deleting the deal.